docs/UserGuide.md

```diff
@@ -4,11 +4,18 @@
   pageNav: 3
 ---
 
-# Knotty Planner 💍🎀
+# Knotty Planner 💍🎀 
 
 Knotty Planner is a **desktop app for wedding planners, optimised for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you are a wedding planner who can type fast, Knotty Planner can make organising weddings a walk in the park!
 
 <!-- * Table of Contents -->
+## Table of Contents
+- [Quick Start](#quick-start)              
+- [Features](#features)         
+- [FAQ](#FAQ)
+- [Known issues](#Known-issues)
+- [Command summary](#Command-summary)  
+
```

Comment:
Could add more links but i think this is fine if not the UG will look too cluttered